webapp/src/js/services/format-data-record.js

```diff
@@ -180,13 +180,36 @@ angular.module('inboxServices').factory('FormatDataRecord',
               });
     };
 
-    var formatDate = function(settings, date) {
+    var capitalizeWord = function(str) {
+      return str.charAt(0).toUpperCase() + str.slice(1);
+    };
+
+    var fromNowOrToday = function(momento, today) {
+      var startOfDay = moment().startOf('day');
+      var duration = moment.duration(startOfDay.diff(momento));
+      if(duration.asHours < 24) {
+        return capitalizeWord(today);
+      }
+      return null;
+    };
+
+    var formatDate = function(settings, date, locale, field) {
       if (!date) {
           return;
       }
       var m = moment(date);
+
+      var fromNow = m.fromNow();
+      if(field === 'child_birth_date') {
+        //Daily precision for child birth date
+        var today = fromNowOrToday(m, translate(settings, 'today', locale));
+        if(today) {
+          fromNow = today;
+        }
```

Comment:
We have a service which does the `fromNowOrToday` bit for you. Use this instead:

```js
FormatDate.age(m, { suffix: true, humanize: true });
```

Play with the options till you get it working the way you want. This will keep this date consistent with everywhere else in the app.